code/game/turfs/turf.dm

```diff
@@ -70,16 +77,29 @@
 	log_startup_progress(" Smoothed atoms in [stop_watch(watch)]s.")
 	return 1
 
-/turf/Destroy()
-// Adds the adjacent turfs to the current atmos processing
+/turf/Destroy(force)
+	. = QDEL_HINT_IWILLGC
```

Comment:
You can't really delete turfs---one instance is invariably replaced with another